cmd/flags.go

```diff
@@ -7,19 +7,21 @@ import (
 )
 
 const (
-	flagConfigPath    = "config"
-	flagVerbose       = "verbose"
-	flagLogLevel      = "log-level"
-	flagJSON          = "json"
-	flagMetricsPort   = "metrics-port"
-	flagFlushInterval = "flush-interval"
-	flagFlushOnlyMode = "flush-only-mode"
+	flagConfigPath     = "config"
+	flagVerbose        = "verbose"
+	flagLogLevel       = "log-level"
+	flagJSON           = "json"
+	flagMetricsAddress = "metrics-address"
+	flagMetricsPort    = "metrics-port"
+	flagFlushInterval  = "flush-interval"
+	flagFlushOnlyMode  = "flush-only-mode"
 )
 
 func addAppPersistantFlags(cmd *cobra.Command, a *AppState) *cobra.Command {
 	cmd.PersistentFlags().StringVar(&a.ConfigPath, flagConfigPath, defaultConfigPath, "file path of config file")
 	cmd.PersistentFlags().BoolVarP(&a.Debug, flagVerbose, "v", false, fmt.Sprintf("use this flag to set log level to `debug` (overrides %s flag)", flagLogLevel))
 	cmd.PersistentFlags().StringVar(&a.LogLevel, flagLogLevel, "info", "log level (debug, info, warn, error)")
+	cmd.PersistentFlags().String(flagMetricsAddress, "localhost", "customize Prometheus metrics host address, this can be used in conjunction with `metrics-port` to adjust entire endpoint")
```

Comment:
```suggestion
	cmd.PersistentFlags().String(flagMetricsAddress, "localhost", "customize Prometheus metrics address, this can be used in conjunction with `metrics-port` to adjust the entire endpoint")
```